src/behaviors/loginPrompt.js

```diff
@@ -1,35 +1,42 @@
 export default {
-  /**
-   * Init login prompt observer
-   * Called by Polymer on attach
-   * @return {undefined}
-   */
-  attached() {
-    this._promptLoginOnEditable();
+  properties: {
+
+    loginPrompt: {
+      type: Boolean,
+      observer: '_promptLoginOnEditable'
+    }
+
   },
 
   /**
    * Observer to prompt for login if editable && !authed
    * @return {undefined}
    */
-  _promptLoginOnEditable() {
-    let simplaLogin = this.$['login'],
+  _promptLoginOnEditable(loginPrompt) {
+    let { _simplaObservers: observers } = this,
+        simplaLogin = this.$['login'],
         promptLogin = (editable) => {
           if (editable && !this._authenticated) {
-            simplaLogin.prompt()
-              .then(loggedIn => Simpla.editable(loggedIn));
+            simplaLogin.prompt().then(loggedIn => {
+              Simpla.editable(loggedIn)
+            });
           }
         },
         initialPrompt = () => {
+          let shouldPrompt = Simpla.getState('editable') && loginPrompt;
+
           if (typeof simplaLogin.prompt === 'function') {
-            promptLogin(Simpla.getState('editable'));
+            promptLogin(shouldPrompt);
```

Comment:
This shouldn't be doing the checks for if it's a function yet - it shouldn't _know_ that `promptLogin` requires `simplaLogin.prompt` to be a function. Instead this check should be done inside the `promptLogin` function. 

Also using the `setTimeout` is pretty hairy - just assumes it'll be ok in 10ms. Ideally the `prompt` should just take a prop like `open` or something which can be set even if the element isn't upgraded yet. Or, do a `setInterval` which gets cleared once `prompt` exists on the function.